package.json

```diff
@@ -32,5 +32,8 @@
     "text"
   ],
   "author": "olizilla",
-  "license": "BSD"
+  "license": "BSD",
+  "engines" : { 
+    "node" : ">=0.12" 
```

Comment:
@olizilla,

Interesting read ... I still don't understand why the build was failing originally with node **0.10** ... but now it is passing, so I have tweaked the `engines` definition to reflect  to what I understand is history. Hopefully i understood it correctly ... still a bit confused, hopefully I can solve that with a Friday 🍺